src/main/java/uk/gov/justice/digital/delius/transformers/ProbationAreaTransformer.java

```diff
@@ -21,6 +21,10 @@ public ProbationAreaTransformer(InstitutionTransformer institutionTransformer) {
         this.institutionTransformer = institutionTransformer;
     }
 
+    public ProbationAreaTransformer() {
+        this.institutionTransformer = new InstitutionTransformer();
```

Comment:
Nice to see this, I hate all this Spring injection of transformers that is just a mess to test

Reply:
Yeah I chewed this over for a bit as it was all getting very messy with dependencies getting passed down multiple levels. It produced a conflict between dependency inversion and the law of Demeter, strict dependency inversion meant passing ridiculous numbers of arguments, most of which were solely to be passed on in turn. I figured for testing we should only ever be concerned with immediate collaborators but we still need to be able to refactor over-sized transformers without breaking existing tests so thats where the default constructors come in.

Reply:
Yeah, given transformers tend to be pure functions there is really no need to mock out. I did make a horrendous mistake where we have some Transformers for updates that do reference data lookups. I regret that now and need to find a better way